cross-platform/react-app/package.json

```diff
@@ -128,5 +128,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "relativeDependencies": {
```

Comment:
I feel that this PR needs an update to cross-platform/react-app/REAME.md.

Reply:
I've updated the README. The instructions are significantly easier now.